EosPayload/drivers/ping_driver.py

```diff
@@ -0,0 +1,96 @@
+import time
+from enum import Enum, unique
+import logging
+
+from EosLib import Device, Priority, Type
+from EosLib.packet.data_header import DataHeader
+from EosLib.packet.packet import Packet
+from EosPayload.lib.driver_base import DriverBase
+from EosPayload.lib.mqtt import Topic
+
+"""
+    This driver is just software and doesn't collect data.
+    It does two functions:
+    (1) Emits regular pings to the ground station
+    (2) Replies to ping commands from the ground station
+    
+    Command syntax:
+        `PING [optional identifier]`
+        `ACK [optional identifier from PING]`
+        `ERR <message>`
+"""
+
+
+class PingDriver(DriverBase):
+
+    @unique
+    class Commands(str, Enum):
+        PING = "PING"
+        ACK = "ACK"
+        ERR = "ERR"
+
+    @staticmethod
+    def get_device_id() -> Device:
+        return Device.MISC_RADIO_1
+
+    @staticmethod
+    def get_device_name() -> str:
+        return "ping-driver"
+
+    def device_command(self, logger: logging.Logger) -> None:
+        self._mqtt.user_data_set({'logger': logger})
+        self._mqtt.register_subscriber(Topic.PING_COMMAND, self.ping_reply)
+        counter = 0
+        while True:
+            self.ping_ground(counter, logger)
+            counter = counter + 1
+            time.sleep(60)
+
+    @staticmethod
+    def ping_reply(client, user_data, message):
+        packet = None
+        try:
+            packet = Packet.decode(message.payload)
+        except Exception as e:
+            user_data['logger'].error(f"failed to decode packet sent to {Topic.PING_COMMAND.value}: {e}")
+            return
+
+        command, param = packet.body.decode('utf8').split(' ', 1)
+        if command not in [cmd.value for cmd in PingDriver.Commands]:
+            user_data['logger'].warning(f"received invalid command '{command}'"
+                                        f" from device '{packet.data_header.sender}'")
+            response_command = PingDriver.Commands.ERR.value + f" invalid command '{command}': '{str(packet.body)}'"
```

Comment:
My concern here is that if command is super long it could potentially cause a fatal error on encode, but maybe that's a problem that should be fixed in EosLib (truncate instead of error on a too-long body or something)?

Reply:
i think it should probably be on the driver writer to do this because sometimes truncation will corrupt the packet so erroring out would be better

Reply:
this line of code is wrong anyway

Reply:
Might be worth defining a static convenience function in EosLib though.  Like
```python
def Packet.str_encode_truncate(string: str, encoding: str) -> bytes:
```